portfolio/src/main/webapp/comments.js

```diff
@@ -46,12 +48,23 @@ async function getCommentsList() {
 * @param {String} text
 * @param {HTMLElement} list
 */
-function addToList(text, list) {
-  const liElement = document.createElement('li');
-  liElement.innerText = text;
-  list.appendChild(liElement);
+function addToList(commentObj, list) {
+  const comment = document.createElement('li');
+  comment.innerText = commentObj.text;
+
+  const deleteButtonElement = document.createElement('button');
+  deleteButtonElement.innerText = 'Delete';
+  deleteButtonElement.addEventListener('click', () => {
+    deleteCommentFromDB(commentObj);
+    comment.remove();
+    getCommentsList();
```

Comment:
Do we need the `comment.remove()` and `getCommentsList()`? As I understand it the first call removes the object from the DOM, and the second reloads the info from the database with the comment removed, so are they doing the same thing?

Reply:
sometimes, there is a lag in the removal and addition of comments.

Reply:
I don't really know why it happens
```suggestion
```